fenics/environment.yml

```
@@ -0,0 +1,8 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - mshr =2019.1.0
+  - pyiron =0.3.21
+  - matplotlib =3.3.3
+  - seaborn =0.11.0
```

```suggestion
  - matplotlib
  - seaborn
```
Do we want to fix a particular version for every package or only for the primary ones?

---

My idea was to have the version of all packages fixed for reproducibility. However, we can limit this to the major packages.

---

At least that is what we have for the other containers. If you want to have all versions fixed and take care of the updates that is fine for me as well.

---

I removed them.